luigi/worker.py

```diff
@@ -315,6 +315,18 @@ def __init__(self, scheduler=None, worker_id=None, worker_processes=1, assistant
         self._task_result_queue = multiprocessing.Queue()
         self._running_tasks = {}
 
+        # Stuff for execution_summary
+        self._add_task_history = []
```

Comment:
I keep mentioning it because so far my mentions haven't been answered at all :)

I thought that it had dependencies and statuses as well, not just task objects...